kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -45,6 +45,8 @@ import elapsedTime from '../views/elapsed-time';
 import pointsIcon from '../views/points-icon';
 import authMessage from '../views/auth-message';
 import kBreadcrumbs from '../views/k-breadcrumbs';
+import kCheckbox from '../views/k-checkbox';
+import kRadio from '../views/k-radio';
```

Comment:
could we call it `k-radiobutton`?

"Radio" sounds like it could be some kind of audio player widget.